giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/service/impl/IdentityProviderRegistry.java

```diff
@@ -34,8 +34,16 @@ public void init() {
     /* (non-Javadoc)
      * @see edu.asu.giles.service.impl.IIdentityProviderRegistry#addProvider(java.lang.String)
      */
+    /**
+     * register a new provider with a given authorization type
+     * e.g authorization type 'mitreidconnect_accessToken' has providerId 'mitreidconnect' 
```

Comment:
This is confusing. The caller wouldn't pass "mitreidconnect_accessToken", would they?